src/components/radio-buttons/radio-buttons--tile.njk

```diff
@@ -17,5 +17,10 @@
       <input class="usa-radio__input usa-radio__input--tile" id="historical-carver-2" type="radio" name="historical-figures-2" value="george-washington-carver" disabled>
       <label class="usa-radio__label" for="historical-carver-2">George Washington Carver</label>
     </div>
+      <div class="usa-radio">
+      <input class="usa-radio__input usa-radio__input--tile" id="historical-marshall-2" type="radio" name="historical-figures" value="thurgood-marshall" disabled checked>
```

Comment:
```suggestion
        <input class="usa-radio__input usa-radio__input--tile" id="historical-marshall-2" type="radio" name="historical-figures-2" value="thurgood-marshall" disabled checked>
```

Reply:
@jeffffeffff The `name="historical-figures-2"` change still isn't done